yesod-static/Yesod/Static.hs

```diff
@@ -273,6 +276,43 @@ staticFilesList dir fs =
 publicFiles :: FilePath -> Q [Dec]
 publicFiles dir = mkStaticFiles' dir False
 
+-- | Similar to 'staticFilesList', but takes a manifest mapping
+-- unmunged names to fingerprinted file names.
```

Comment:
Can you add a `@since` comment here and below, as well as a `ChangeLog.md` entry and bumping the minor version in the cabal file?